source/trunk/Esf.Web/App_Start/RouteConfig.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+using System.Web.Routing;
+
+namespace Esf.Web
```

Comment:
I guess we'll setup less, typescript and the website folder structure as a separate task? It must be definitely a separate checkin atleast.